testprojects/src/python/sources/BUILD

```diff
@@ -1,3 +1,3 @@
 python_library(
-  resources=globs('*.txt'),
+  sources=globs('*.py'),
```

Comment:
The explicit sources arg shouldn't be necessary with implicit_sources. But does this need a second target to depend on?
```
python_library(
  dependencies=[':resources'],
)

resources(
  sources=globs('*.txt'),
)
```